image/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Common builder
+FROM golang:1.10-alpine3.7 as builder
+
+#COPY hack/dockerfile/binaries-commits /
```

Comment:
Will we be using the binaries-commits at some point or is it okay to remove them?

Reply:
At some point in the last month or two the way we store the dependencies changed in the tree, and as I bounced back and forth between HEAD and shipped tags I had to keep switching models, so I left this in commented out to make it easier to transition.  Ultimately I think we need to refactor this in moby based on a top-level manifest or something along those lines, but I didn't want to make this dependent on that refactoring.